src/tribler-core/tribler_core/modules/tracker_manager.py

```diff
@@ -126,12 +126,20 @@ def get_next_tracker_for_auto_check(self):
         Gets the next tracker for automatic tracker-checking.
         :return: The next tracker for automatic tracker-checking.
         """
-        tracker = self.tracker_store.select(lambda g: str(g.url) not in [u'no-DHT', u'DHT']
-                                            and g.alive
-                                            and g.last_check + TRACKER_RETRY_INTERVAL <= int(time.time())
-                                            and str(g.url) not in self.blacklist)\
-            .order_by(self.tracker_store.last_check).limit(1)
+        tracker = (
```

Comment:
Be *very* careful with PonyORM queries! Please, check twice that this thing works as it should! Do not make the change, if the method is not tested.

Reply:
Will undo the change.

Reply:
Undid the change.